src/components/Tabs/Tabs.vue

```diff
@@ -21,22 +21,22 @@ div(:class="styles.Outer")
           data-tabs-focus-catchment,
         )
           Tab(
-            v-for="_tab, index in tabData",
-            v-bind="tabs[index]",
+            v-for="tabIndex in tabData",
```

Comment:
I did a quick check and tabData is an array and changing from `_tab, index` to `tabIndex` might be wrong. Can you give me more details on why you did that?

Reply:
Hello,

From my understanding, and correct me if i am wrong, tabData if an array of numbers that represent each index that we want to show as a tab, for example if we select the tab with the name "content-29" (previous picture from the original post), we want to show it as the last tab element.

![image](https://github.com/user-attachments/assets/b4122321-69a9-401a-9403-e2b872630182)